easybuild/easyconfigs/n/NAG/NAG-26-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'Binary'
```

Comment:
@SimonPinches You can use the `PackedBinary` easyblock instead, which simplifies things a bit below (no need to unpack yourself via `preinstallopts`.

Reply:
I've tried to make this work with the `<PackedBinary>` easyblock but this seems to ignore the `<install_cmd>` so I've failed to get it to work...